### PR TITLE
Demo site created for EUID JS SDK CSTG 

### PR DESCRIPTION
**Pull request by ashleysmithTTD**
Created a new demo site for EUID JS SDK CSTG and took out any existing logic for EUID that was in UID2 JS SDK for CSTG.  Maintaining a separation of anything UI-related between EUID and UID2

### CODE REVIEW

**Comment by ashleysmithTTD** [comment]
Test Plan:

Both pages - UID2 and EUID need to be tested since I changed code in both

Once in the branch:

Go to "Run and Debug" in VSCode and select, "Launch CSTG (Chrome)" to test UID2 

Go to "Run and Debug" in VSCode and select "Launch CSTG EUID (Chrome)" to test EUID

**Review by alex-yau-ttd — APPROVED** [review]

File: examples/cstg/html/index.html [hunk 1]
```diff
@@ -14,89 +14,52 @@
         serverPublicKey: '{{ SERVER_PUBLIC_KEY }}',
       };
 
-      let callbackCounter = {
```
Comment (alex-yau-ttd) [inline examples/cstg/html/index.html:17]: Should we remove `<script defer src="{{ EUID_JS_SDK_URL }}"></script>` from this file?

File: examples/cstg-euid/html/index.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>EUID Publisher Client-Side Integration Example using UID2 JavsScript SDK</title>
+    <link rel="stylesheet" type="text/css" href="/stylesheets/app.css" />
+    <link rel="shortcut icon" href="/images/favicon.png" />
+    <script defer src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script defer src="{{ UID2_JS_SDK_URL }}"></script>
```
Comment (alex-yau-ttd) [inline examples/cstg-euid/html/index.html:9]: Is this line needed?

Comment (ashleysmithTTD) [reply examples/cstg-euid/html/index.html]: Removed

File: examples/cstg-euid/html/index.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      }
+
+      window.__euid = window.__euid || {};
+      window.__euid.callbacks = window.__euid.callbacks || [];
+
+      window.__euid.callbacks.push(onEuidIdentityUpdated);
+      window.__euid.callbacks.push((eventType, payload) => {
+        if (eventType === 'SdkLoaded') {
+          window.__euid.init({
+            baseUrl: '{{ UID2_BASE_URL }}',
```
Comment (lionell-pack-ttd) [inline examples/cstg-euid/html/index.html:75]: You'll need to provide EUID values for this and the keys, but I don't think we need them to be provided by the server - the whole point of CSTG is that the server isn't needed :) so they can be hard-coded here.

File: examples/cstg-euid/html/index.html [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    <meta charset="UTF-8" />
+    <title>EUID Publisher Client-Side Integration Example using UID2 JavsScript SDK</title>
+    <link rel="stylesheet" type="text/css" href="/stylesheets/app.css" />
+    <link rel="shortcut icon" href="/images/favicon.png" />
+    <script defer src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script defer src="{{ EUID_JS_SDK_URL }}"></script>
+    <script>
+      const clientSideIdentityOptions = {
+        subscriptionId: '{{ SUBSCRIPTION_ID }}',
+        serverPublicKey: '{{ SERVER_PUBLIC_KEY }}',
```
Comment (lionell-pack-ttd) [inline examples/cstg-euid/html/index.html:13]: You'll need EUID values for these (see my comment on the Prebid.js example PR) but they can just be included here - they don't need to be server-side config values like this.

**Comment by lionell-pack-ttd** [comment]
One note on this about deployment: because it's set up to run as a container, we'll need k8s resources and a DNS entry for this to work. But because there's no server-side secrets for CSTG, the container isn't needed - it might be easier to take the new EUID-related files and move them to the uid2-docs repo so it can just be hosted at unifiedid.com, just like the Prebid.js CSTG example you did.

**Pull request closed without merging**